website_snippet_country_dropdown/views/snippets.xml

```diff
@@ -52,13 +51,14 @@
 
 <template id="country_dropdown" name="Country Dropdown">
     <t t-set="countries" t-value="request.env['res.country'].search([])"/>
-    <div id="snippet_container" class="input-group" t-ignore="true">
-        <div class="input-group-btn">
+    <t t-set="default_country" t-value="countries[:1]"/>
```

Comment:
If I pass `default_country` in the `t-call`, would it get the one I passed or `countries[:1]`? Shouldn't this value be `default_country or countries[:1]`?